inc/Engine/Admin/Settings/Page.php

```diff
@@ -916,7 +926,25 @@ private function assets_section() {
 					'input_attr'        => [
 						'disabled' => get_rocket_option( 'delay_js' ) ? 0 : 1,
 					],
-					'helper'            => DelayJSSettings::exclusion_list_has_default() ? $delay_js_found_list_helper : $delay_js_list_helper,
+					'items'             => $this->delayjs_sitelist->prepare_delayjs_ui_list(),
+				],
+				'delay_js_exclusions'          => [
+					'type'              => 'custom_textarea',
+					'label'             => __( 'My Scripts', 'rocket' ),
+					'description'       => __( 'Specify URLs or keywords that can identify inline or JavaScript files to be excluded from delaying execution (one per line).', 'rocket' ),
+					'container_class'   => [
+						'wpr-field--children',
+					],
+					'parent'            => 'delay_js',
+					'section'           => 'js',
+					'page'              => 'file_optimization',
+					'default'           => [],
+					'sanitize_callback' => 'sanitize_textarea',
+					'input_attr'        => [
+						'disabled' => get_rocket_option( 'delay_js' ) ? 0 : 1,
+					],
+					// 'helper'            => DelayJSSettings::exclusion_list_has_default() ? $delay_js_found_list_helper : $delay_js_list_helper,
```

Comment:
```suggestion
					// 'helper'            => DelayJSSettings::exclusion_list_has_default() ? $delay_js_found_list_helper : $delay_js_list_helper,
```

Is this going to be used or not?

Reply:
@piotrbak this part was there to show a helper message in two cases:
1. If the default exclusion list is there so we show `Internal scripts are excluded by default to prevent issues. Remove them to take full advantage of this option.<br>If this causes trouble, restore the default exclusions, found here`
2. If users changed the list, show the msg: `If you have problems after activating this option, copy and paste the default exclusions to quickly resolve issues:<br><pre><code>%1$s</code></pre><br>Also, please check our %2$sdocumentation%3$s for a list of compatibility exclusions.`

I don't think we still need this, what do u think?

Reply:
@engahmeds3ed Yes, you're correct